src/adapter/driver/api/routes/clienteRouter.ts

```diff
@@ -0,0 +1,18 @@
+import { Router } from "express";
+import { clienteController } from "@controllers/index";
+
+export function makeClienteRouter(): Router {
+    const clienteRouter = Router();
+
+    clienteRouter.post("/", async (req, res) =>
+        clienteController.post(req, res),
+    );
```

Comment:
sugestão (pode ignorar se quiser):
```suggestion
    );
   
```